src/Umbraco.Web.UI.Client/src/views/users/views/users/users.html

```diff
@@ -211,9 +211,9 @@
                 <div class="umb-table-head">
                     <div class="umb-table-row">
                         <div class="umb-table-cell umb-user-table-col-avatar not-fixed" style="width: 70px; padding:10px 15px;">
-                            <a href="" style="text-decoration: none;" ng-click="vm.selectAll()">
+                            <button type="button" class="btn-reset umb-checkmark__action" style="outline: none;" ng-click="vm.selectAll()">
```

Comment:
@BatJan could `umb-outline` or `btn-outline` class be used here?

Reply:
@bjarnef I'm not sure - Maybe. But I use this temporary class to fix some underlying states too - This fix is temporary untill I have a stab at refactoring the `<umb-checkmark>` anyway. But that is for another PR. Once I have a look at that it might be worth exploring the `umb-outline` or `btn-outline` classes 👍 😃